### ISSUE TITLE
Can we minimize the negative log-likelihood of target words without pointer_gen mode?

### ISSUE THREAD

**Issue by Sanqiang** [abisee/pointer-generator#54]
Without pointer_gen mode, the loss is calculated by tf.contrib.seq2seq.sequence_loss. However, with pointer_gen mode, the loss minimizes the negative log-likelihood of target words.  The loss makes sense for me and but I just wonder whether we cannot also minimize the negative log-likelihood of target words without pointer_gen mode?

**Comment by abisee** [comment]
Though it may not be obvious from the name, negative log likelihood and [cross entropy](https://www.tensorflow.org/api_docs/python/tf/contrib/seq2seq/sequence_loss) are the same in this instance.

**Comment by Sanqiang** [comment]
Exactly, after re-read the code, it maximizes the probability from vocab_dists (generated from decoder output) with the index from the target (from the ground truth). So it is equivalent with cross-entropy.
